onnx/defs/printer.h

```diff
@@ -38,4 +38,13 @@ std::ostream& operator<<(std::ostream& os, const GraphProto& graph);
 
 std::ostream& operator<<(std::ostream& os, const FunctionProto& fn);
 
+std::ostream& operator<<(std::ostream& os, const ModelProto& model);
+
+template <typename ProtoType>
+std::string ProtoToString(const ProtoType& proto) {
```

Comment:
Why is this function added given we have all `operator<<`s above?

Reply:
It's just a convenient short-hand. Further, for Python, we just expose this functionality instead of the operator<<.